frontend/src/metabase/visualizations/components/ObjectDetail/utils.ts

```diff
@@ -143,8 +147,16 @@ export const getActionItems = ({
   onUpdate: (action: WritebackAction) => void;
 }) => {
   const actionItems = [];
-  const deleteAction = actions.find(isValidImplicitDeleteAction);
-  const updateAction = actions.find(isValidImplicitUpdateAction);
+  /**
+   * Public actions require an additional endpoint which is out of scope
+   * of Milestone 1 in #32320 epic.
+   *
+   * @see https://github.com/metabase/metabase/issues/32320
+   * @see https://metaboat.slack.com/archives/C057T1QTB3L/p1689845931726009?thread_ts=1689665950.493399&cid=C057T1QTB3L
+   */
+  const privateActions = actions.filter(action => !action.public_uuid);
```

Comment:
I don't think this is correct. If you make either edit or delete action public. You won't be able to use those action in the model detail anymore.

![Screenshot 2023-07-24 at 6 31 01 PM](https://github.com/metabase/metabase/assets/1937582/a993a2cd-e235-4476-a6ed-d62e2c8c0edc)

![image](https://github.com/metabase/metabase/assets/1937582/0f4b542e-7adb-4204-82aa-443059f03f88)

Reply:
> I don't think this is correct. If you make either edit or delete action public. You won't be able to use those action in the model detail anymore.

Public actions require a separate endpoint and it's been put on hold for now, please see https://metaboat.slack.com/archives/C057T1QTB3L/p1689801698386559?thread_ts=1689665950.493399&cid=C057T1QTB3L

Reply:
I think I understand that in a different way.

I think:
1. A normal model should always show edit/delete if actions are enabled, basic actions are enabled.
2. A model can be put on a dashboard, though I couldn't find a way to open object detail from models on a dashboard.
3. A model action could be made public. But that shouldn't take away the ability to edit/delete when not viewed publicly (point 1)
4. I think if we have a way to view object detail publicly e.g. via public dashboards then that's what is not included in the scope of the project.

cc @cdeweyx for clarification.

Reply:
> 2. A model can be put on a dashboard, though I couldn't find a way to open object detail from models on a dashboard.

Yeah, it's not possible, see [product doc](https://www.notion.so/metabase/Allow-updates-and-deletes-on-model-object-detail-views-f05b298c3aba4a6ca883b2738d86824d?pvs=4#75d2dac5e2c7477faba202b6e803d922).

Reply:
What Kamil said